lib/oplogjam/operators/intermediate.rb

```diff
@@ -36,7 +36,7 @@ def set_field(path, value)
       end
 
       def set_index(path, value)
-        tree[path] = IndexAssignment.new(path, value)
+        tree[path] = FieldAssignment.new(path, value)
```

Comment:
For logging, could we just add a `puts` after line 20? With the path and the value. Just to know how often we encounter the situation we are trying to fix.

Reply:
It results in:

```log
NUMERIC INDEX DETECTED: path => ["a", "7"], value: 7
NUMERIC INDEX DETECTED: path => ["a", "8"], value: 8
NUMERIC INDEX DETECTED: path => ["a", "9"], value: 9
NUMERIC INDEX DETECTED: path => ["a", "10"], value: 10
NUMERIC INDEX DETECTED: path => ["a", "11"], value: 11
```

WDYT?

Reply:
Yes, I think that this is enough

Reply:
Done: https://github.com/altmetric/oplogjam/pull/1/commits/0848cda89fa659f2bd5b6e549d6099f8b7c821d2